spectrum/checks.py

```diff
@@ -669,6 +669,35 @@ def recommend(self, manuscript='', subject='', keywords='', abstract=''):
             raise UnrecoverableError(response)
         return response.json()
 
+class ObserverCheck:
+    def __init__(self, host):
+        self._host = host
+
+    def latest_article(self, id):
+        template = "%s/report/latest-articles"
+        url = template % self._host
+        return _poll(
+            lambda: self._is_present(url, id),
+            "article with id %s at %s",
+            id, url
+        )
+
+    def _is_present(self, url, id):
+        response = requests.get(url)
+        LOGGER.debug("Loaded %s (%s)", url, response.status_code)
+        if response.status_code > 299:
+            raise UnrecoverableError(response)
+        soup = BeautifulSoup(response.content, "lxml-xml")
+        target_guid = "https://dx.doi.org/10.7554/eLife.%s" % id
```

Comment:
zero-padding shouldn't be necessary for the id (msid) here, right?

Reply:
Don't think so, because they are all very long.